static/js/search_suggestion.js

```diff
@@ -162,6 +162,80 @@ function get_private_suggestions(all_people, operators, person_operator_matches)
     return suggestions;
 }
 
+function get_group_suggestions(all_people, operators) {
+    if (operators.length === 0) {
+        return [];
+    }
+
+    if ((operators[0].operator === 'is') && (operators[0].operand === 'private')) {
+        operators = operators.slice(1);
+    }
+
+    if (operators.length !== 1 || operators[0].operator !== 'pm-with') {
+        return [];
+    }
```

Comment:
Right--I used the same approach as `get_private_suggestions` but I agree that the situation could be improved.  I'd be happy to look into this next.